src/main/java/frc/robot/vision/FollowTarget.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.vision;
+
```

Comment:
just see my comments from the TurnToTarget command it is basically the same...

Reply:
Should I inline distanceOutput and turnOutput in FollowTarget too? It becomes kinda long:
`      output.accept(rotationPIDController.calculate(limelight.getAngle()), distancePIDController.calculate(limelight.getDistance()));`

Reply:
and about the subsystem not being necessarily drivetrain, there is any other subsystem that can move in such a way?

Reply:
no, you right you can leave the drivetrain here as it is

Reply:
> Should I inline distanceOutput and turnOutput in FollowTarget too? It becomes kinda long:
> ` output.accept(rotationPIDController.calculate(limelight.getAngle()), distancePIDController.calculate(limelight.getDistance()));`

you right also here. these variables should stay.